src/sigal/gallery.py

```diff
@@ -935,8 +936,23 @@ def remove_files(self, medias):
 
     def process_dir(self, album, force=False):
         """Process a list of images in a directory."""
+        def forcing(a):
+            if force is None:
+                return False
+            if isinstance(force, bool):
+                return force
+            elif len(force) == 0:
+                return True
+            else:
+                for f in force:
+                    if '*' in f or '?' in f:
+                        if fnmatch(a.path, f):
+                            return True
```

Comment:
```suggestion
                        return fnmatch(a.path, f)
```

Reply:
I don't think that works because a False would be an early return